src/proto/grpc/testing/control.proto

```diff
@@ -94,9 +94,12 @@ message ClientConfig {
   repeated string server_targets = 1;
   ClientType client_type = 2;
   SecurityParams security_params = 3;
-  // How many concurrent RPCs to start for each channel.
+  // Maximum limit of how many concurrent RPCs to start for each channel.
```

Comment:
nit: Maximum number of concurrent RPCs in flight for each channel at any given time?